__tests__/components/__snapshots__/NetworkSwitch.test.js.snap

```diff
@@ -633,7 +633,7 @@ exports[`NetworkSwitch renders without crashing 1`] = `
                     }
                   >
                     <div
-                      className="css-k5geue react-select__value-container react-select__value-container--has-value"
+                      className="css-6zqqc react-select__value-container react-select__value-container--has-value"
```

Comment:
Not sure why this changed since I touched no css.. ?

Reply:
it is prob an autogenerated thing related to css modules